From patchwork Sat Nov  7 21:55:07 2009
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [43/75] spider-net: declare MODULE_FIRMWARE
Date: Sat, 07 Nov 2009 11:55:07 -0000
From: Ben Hutchings <ben@decadent.org.uk>
X-Patchwork-Id: 37927
Message-Id: <1257630907.15927.445.camel@localhost>
To: David Miller <davem@davemloft.net>
Cc: Ishizaki Kou <kou.ishizaki@toshiba.co.jp>,
	Jens Osterkamp <jens@de.ibm.com>, netdev <netdev@vger.kernel.org>

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>

---
drivers/net/spider_net.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/drivers/net/spider_net.c
+++ b/drivers/net/spider_net.c
@@ -57,6 +57,7 @@ MODULE_AUTHOR("Utz Bacher <utz.bacher@de
 MODULE_DESCRIPTION("Spider Southbridge Gigabit Ethernet driver");
 MODULE_LICENSE("GPL");
 MODULE_VERSION(VERSION);
+MODULE_FIRMWARE(SPIDER_NET_FIRMWARE_NAME);
 
 static int rx_descriptors = SPIDER_NET_RX_DESCRIPTORS_DEFAULT;
 static int tx_descriptors = SPIDER_NET_TX_DESCRIPTORS_DEFAULT;
